components/tests/ui/README.md

```diff
@@ -48,14 +48,14 @@ If you are running the tests on Mac OS X, you can install with the following
 brew install geckodriver
 ```
 
+NB: this took too long for me, and manually installing geckodriver
```

Comment:
Maybe change this to "If installing takes too long, ..."

Reply:
Fixed in b4700164af